components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```diff
@@ -263,15 +276,13 @@ def create_results_cache_indices(
         "--rm",
         "--name", container_name,
         "--log-driver", "local",
-        "-e", f"PYTHONPATH={clp_site_packages_dir}",
         "-u", f"{os.getuid()}:{os.getgid()}",
     ]
     # fmt: on
+    necessary_env_vars = [f"PYTHONPATH={clp_site_packages_dir}"]
```

Comment:
let's not name it as "necessary" unless there's an optional counter part for the same component.
I understand that you copied the "necessary" from necessary_mounts. Tbh, I feel the name for mount is also confusing but let's not touch it in this PR.